### PR TITLE
chore(deps): update dependencies

### PR DESCRIPTION
**Pull request by anehx**
Closes #240 
Closes #241 
Closes #242  
Closes #237 
Closes #236 
Closes #235 
Closes #233 
Closes #231
Closes #229
Closes #227 
Closes #222 
Closes #219

### CODE REVIEW

File: addon/components/uk-modal.js [hunk 1]
```diff
@@ -60,13 +60,19 @@ export default Component.extend({
...
-    });
+    scheduleOnce("afterRender", this.initVisible);
   },
 
   willDestroyElement() {
     this.modal.$el.parentNode.removeChild(this.modal.$el);
     this.set("modal", null);
+  },
+
+  initVisible() {
```
Comment (fkm) [inline addon/components/uk-modal.js]: Not that I'm against this change. But why did you do this? Does this fix anything technical?

Comment (anehx) [reply addon/components/uk-modal.js]: https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/no-incorrect-calls-with-inline-anonymous-functions.md This rule is now enabled per default

Comment (fkm) [reply addon/components/uk-modal.js]: Thank you!

**Comment by anehx** [comment]
https://github.com/ember-cli/ember-cli-htmlbars/issues/312 This is the reason for the failing build. Will fix as soon as they released a fix..

**Pull request merged**